### PR TITLE
Adds New Catalog endpoint description and example

### PR DESCRIPTION
**Pull request by susiirwin**

### CODE REVIEW

File: app/docs-v3/auth-api.html [hunk 1]
```diff
@@ -197,6 +204,44 @@ <h4>Examples</h4>
...
+				</div>
+				<pre class="prettyprint">
+Response: 200
+{
+  userId: 9999,
+  userDisplayName: "Joe Black",
+  userEmail: "joe.black@yourorganization.com",
+  userFirstName: "Joe",
+  userLastName: "Black",
+  dashboardUrl: "http://yourorg.mindflash.com/CreateCookie.aspx?sessionID2=AcAVGBh34343SD"
```
Comment (zadevictor) [inline app/docs-v3/auth-api.html]: small correct: the url format will be : https://yourorg.mindflash.com/d=.....

File: app/docs-v3/auth-api.html [hunk 2]
```diff
@@ -197,6 +204,44 @@ <h4>Examples</h4>
...
+  catalogs: [
+    {
+      id: 123,
+      name: "How to Hang a Door",
+      url: "https://yourorg.mindflash.com/CreateCookie.aspx?sessionID2=B4dawfjkCVCFct"
```
Comment (zadevictor) [inline app/docs-v3/auth-api.html]: ditto

File: app/docs-v3/auth-api.html (hunk 1 above)
Comment (susiirwin) [reply app/docs-v3/auth-api.html]: thanks!!!

File: app/docs-v3/auth-api.html (hunk 2 above)
Comment (susiirwin) [reply app/docs-v3/auth-api.html]: @zadevictor - take a look :) make sure I got them this time :)

**Pull request merged**